clean_project/index.py

```diff
@@ -0,0 +1,85 @@
+"""
+Create a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import clean_file
+import psycopg2
+import yaml
+
+from collections import defaultdict
+from ConfigParser import SafeConfigParser
+
+
+def load_config_file():
+    config = SafeConfigParser()
+    config.read('test_data/config.ini')
```

Comment:
Would you want that users define a config path when they use index.py in command line?